src/extensions/service-vendors/twilio/index.js

```diff
@@ -633,44 +633,46 @@ export async function getContactInfo({
     return {};
   }
   const twilio = await exports.getTwilio(organization);
-  const types = ["carrier"];
+  const types = { fields: "line_type_intelligence" };
   if (lookupName) {
     // caller-name is more expensive
-    types.push("caller-name");
+    types.fields = ("line_type_intelligence,caller-name");
```

Comment:
should this be an array instead of parentheses?

Reply:
Oddly enough, no: https://www.twilio.com/docs/lookup/v2-api#code-lookup-with-data-packages
Twilio wants `types.fields` as a comma separated string. That parentheses is awkward though